server/people.js

```diff
@@ -19,6 +19,24 @@ const getPerson = field => async (client, id) => {
   return rows[0];
 };
 
+const addPerson = async (client, email, password) => {
+  const result = await client.query(
+    'INSERT INTO "Dootman"(id, email, password) VALUES ($1, $2, $3)',
+    [Date.now(), email, password]
```

Comment:
We need to actually hash the password. So import bcrypt and replace `password` with `await bcrypt.hash(password, 10)`

Reply:
this is again my bad.. bcrypt was throwing some error that was not letting me run the server. It kept happening over and over. I tried npm install to get us up to date, looking at the projects dependencies and trying to figure out why. 

So I ended up just deleting the references to bcrypt for the moment so I could at least get working on the "delete user" functionality. I was going to put them back and try to figure out what was wrong later today or monday.